components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -49,22 +52,33 @@ FilecoinKeyring::Type FilecoinKeyring::type() const {
 
 std::string FilecoinKeyring::ImportFilecoinBLSAccount(
     const std::vector<uint8_t>& private_key,
-    const std::vector<uint8_t>& public_key,
     const std::string& network) {
-  if (private_key.empty() || public_key.empty()) {
+#if BUILDFLAG(ENABLE_RUST_BLS)
+  if (private_key.empty()) {
```

Comment:
is this expected to happen? if not i'm wondering if this should throw an error instead of returning empty string.

Reply:
yes, users can paste bad private keys and underlying libraries may return empty public keys